React/tic-tac/src/redux/auth/reducer.js

```diff
@@ -0,0 +1,15 @@
+import actions from '@redux/auth/actions';
+
+function reducerAuth(state = false, action) {
+  switch (action.type) {
+    case actions.logInTrue().type:
```

Comment:
I think it's easier if you define just a plain object that stores the action types you're going to use.